conjurapi/client.go

```diff
@@ -114,6 +114,61 @@ func NewClientFromEnvironment(config Config) (*Client, error) {
 		return NewClientFromTokenFile(config, authnTokenFile)
 	}
 
+	authnToken := os.Getenv("CONJUR_AUTHN_TOKEN")
```

Comment:
Mayne this code can be broken to few functions of
* Read JWT Token File
* Build request
* Send Request

Reply:
Please feel free to refactor it :)